internal/cli/dialog/proposals_show_lineage.go

```diff
@@ -0,0 +1,56 @@
+package dialog
+
+import (
+	"fmt"
+
+	"github.com/git-town/git-town/v22/internal/cli/dialog/dialogcomponents"
+	"github.com/git-town/git-town/v22/internal/cli/dialog/dialogcomponents/list"
+	"github.com/git-town/git-town/v22/internal/cli/dialog/dialogdomain"
+	"github.com/git-town/git-town/v22/internal/forge/forgedomain"
+	"github.com/git-town/git-town/v22/internal/messages"
+	. "github.com/git-town/git-town/v22/pkg/prelude"
+)
+
+const (
+	proposalsShowLineageTitle = `Proposals Show Lineage`
+	proposalsShowLineageHelp  = `
+How should Git Town share stack lineage in proposals?
+
+Possible Options:
+	- none: (default) Git Town should not share stack 
+					lineage in proposals
+	- cli: Git Town shares or updates stack lineage in proposals
+				anytime your stack lineage changes via the cli
+	- ci: Git Town manages and shows proposal lineage through ci 
+	      integrations, with tools such as https://github.com/git-town/action
```

Comment:
Can you merge these explanations with the ones in the radio list through which the user chooses the option?